test/pdks/gf180mcu_osu_sc/test_dlat.py

```
@@ -0,0 +1,30 @@
+from charlib.characterizer.characterizer import Characterizer
+
+if __name__ == "__main__":
+    # Test gf180mcu_osu_sc 9t dff_1
+    characterizer = Characterizer(**{
+        'lib_name': 'gf180mcu_osu_sc_gp9t3v3__dlat_1_test',
+        'debug': True,
+        'multithreaded': False,
+    })
```

Ruff (0.15.21)

[warning] Unnecessary `dict` kwargs

Remove unnecessary kwargs

(PIE804)